project/SbtMultiJvm.scala

```diff
@@ -372,7 +373,13 @@ object MultiJvmPlugin extends AutoPlugin {
         val connectInput = input && index == 0
         log.debug("Starting %s for %s".format(jvmName, testClass))
         log.debug("  with JVM options: %s".format(allJvmOptions.mkString(" ")))
-        (testClass, Jvm.startJvm(javaBin, allJvmOptions, runOptions, jvmLogger, connectInput))
+        val testClass2Process = (testClass, Jvm.startJvm(javaBin, allJvmOptions, runOptions, jvmLogger, connectInput))
+        if (index == 0) {
+          log.debug("%s for %s 's started as `Controller`, waiting 2 seconds before start clients.".format(jvmName,
+            testClass))
+          TimeUnit.SECONDS.sleep(2)
```

Comment:
This is needed to get it works on linux and mac.

Reply:
I don't like time based waits. On a slow machine, they can stop working. Our CI builds on VMs that can be very slow.

Is there any way that this code could be changed to ping the netty cluster and keep doing so until it gets notification that the cluster is ready?

Reply:
Regarding this point, maybe it makes sense to make this a separate issue to look at after this PR gets merged? The reason I am stating this is because the best way to find out if these time based constants are too low is by seeing how often the tests fail in our nightly github CI action and we can then iterate from there.

Of course most ideal would be to ping the netty cluster until its ready but I don't feel its valid to block the PR over this, for me its critical to get people to test this directly (via snpashots/building locally), hopefully in production to see if there aren't any regressions well before a release happens.